src/container.ts

```diff
@@ -97,6 +228,52 @@ export class EnvValue {
     return new EnvValue(value);
   }
 
+  /**
+   *
+   * Create a value from a field reference.
+   *
+   * @param fieldPath: The field reference.
+   * @param options: Additional options.
+   */
+  public static fromFieldRef(fieldPath: EnvFieldPaths | FieldPaths, options: EnvValueFromFieldRefOptions = {}): EnvValue {
```

Comment:
We generally avoid using type unions in APIs if possible because when cdk8s-plus gets transpiled by JSII to other languages like Java and .NET, users in those languages will not see any typing information (because those languages don't support type unions natively). See https://aws.github.io/jsii/specification/2-type-system/#type-unions for more info.

Perhaps we could combine all of the allowed values into `EnvFieldPaths`? I'm okay with having some redundancy among enums if needed.

Reply:
Hi, have currently merged the two into a EnvFieldPaths.